### PR TITLE
14505 admin groups panel

### PR DESCRIPTION
**Pull request by burzomir**
...

### CODE REVIEW

File: src/views-components/main-app-bar/account-menu.tsx [hunk 1]
```diff
@@ -34,7 +34,6 @@ export const AccountMenu = connect(mapStateToProps)(
                     {getUserFullname(user)}
                 </MenuItem>
                 <MenuItem onClick={() => dispatch(openUserVirtualMachines())}>Virtual Machines</MenuItem>
-                {!user.isAdmin && <MenuItem onClick={() => dispatch(openRepositoriesPanel())}>Repositories</MenuItem>}
```
Comment (kowalczykpawel7) [inline src/views-components/main-app-bar/account-menu.tsx]: usefull code

File: src/views-components/main-app-bar/admin-menu.tsx [hunk 2]
```diff
@@ -35,9 +35,9 @@ export const AdminMenu = connect(mapStateToProps)(
                 <MenuItem onClick={() => dispatch(NavigationAction.navigateToSshKeysAdmin)}>Ssh Keys</MenuItem>
                 <MenuItem onClick={() => dispatch(NavigationAction.navigateToApiClientAuthorizations)}>Api Tokens</MenuItem>
                 <MenuItem onClick={() => dispatch(openUserPanel())}>Users</MenuItem>
+                <MenuItem onClick={() => dispatch(NavigationAction.navigateToGroups)}>Groups</MenuItem>}
                 <MenuItem onClick={() => dispatch(NavigationAction.navigateToComputeNodes)}>Compute Nodes</MenuItem>
                 <MenuItem onClick={() => dispatch(NavigationAction.navigateToKeepServices)}>Keep Services</MenuItem>
                 <MenuItem onClick={() => dispatch(NavigationAction.navigateToLinks)}>Links</MenuItem>
-                <MenuItem onClick={() => dispatch(logout())}>Logout</MenuItem>
```
Comment (kowalczykpawel7) [inline src/views-components/main-app-bar/admin-menu.tsx]: :+1:

File: src/views/groups-panel/groups-panel.tsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
+// Copyright (C) The Arvados Authors. All rights reserved.
+//
+// SPDX-License-Identifier: AGPL-3.0
+
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Grid, Button, Typography } from "@material-ui/core";
+
```
Comment (kowalczykpawel7) [inline src/views/groups-panel/groups-panel.tsx]: white space

File: src/views-components/main-app-bar/account-menu.tsx (hunk 1 above)
Comment (burzomir) [reply src/views-components/main-app-bar/account-menu.tsx]: restored

File: src/views/groups-panel/groups-panel.tsx (hunk 3 above)
Comment (burzomir) [reply src/views/groups-panel/groups-panel.tsx]: done

**Pull request merged**